cartridges/int_adyen_SFRA/cartridge/client/default/js/adyenCheckout.js

```diff
@@ -34,19 +37,24 @@ function renderCardComponent() {
     card = checkout.create('card', {
         // Mandatory fields
         originKey: originKey,
-        loadingContext: loadingContext, // The environment where we should loads the secured fields from
+        environment: loadingContext, // The environment where we should loads the secured fields from
         type: 'card',
         hasHolderName: true,
         holderNameRequired: true,
-        groupTypes: ["bcmc", "maestro", "visa", "mc", "amex", "diners", "discover"],
+        groupTypes: ["bcmc", "maestro", "visa", "mc", "amex", "diners", "discover", "jcb"],
 
         // Events
         onChange: function (state) {
             isValid = state.isValid;
         }, // Gets triggered whenever a user changes input
         onBrand: function (brandObject) {
             $('#cardType').val(brandObject.brand);
-        } // Called once we detect the card brand
+        }, // Called once we detect the card brand
+        onFieldValid: function (data) {
+            if(data.endDigits){
+                $('#cardNumber').val("**********" + data.endDigits);
```

Comment:
why the stars?

Reply:
The maskedCreditCardNumber value in a payment instrument will automatically hide the first digits with stars. If there are no stars provided, the endDigits will end up hidden behind stars based on the logic in SFCC.